File: swea/D3/2805.농작물 수확하기/2805.cpp
#include <iostream>
#include <cmath>
#include <vector>
#include <string>

using namespace std;

int T;

int main(){
  cin >> T;
  for(int t = 1; t <= T; t++){
    int n;
    cin >> n;
    
    vector<string> arr(n);
    for(int i = 0; i < n; i++){
        cin >> arr[i];
    }
    
    int mid = n / 2, ans = 0;

    for(int i = 0; i < n; i++){
      int diff = abs(mid - i);

      for(int j = diff; j < n - diff; j++){
        ans += arr[i][j] - '0';
      }
    }

    cout << '#' << t << " " << ans << '\n';
  }
}

File: swea/D3/2805.농작물 수확하기/2805.py
T = int(input())
for tc in range(1, T+1):
    n = int(input())
    arr = [list(map(int, input().strip())) for _ in range(n)]

    mid = n // 2
    ans = 0
    for i in range(n):
        diff = abs(mid - i)
        ans += sum(arr[i][diff: n - diff])
    
    print(f'#{tc} {ans}')